### PR TITLE
Adding documentation on “How to add a layer definition”

### LINKED ISSUE #124: Add documentation on “How to add a layer definition”
related to #116

### PR DESCRIPTION
**Pull request by hn437**
### Description
Adding a documentation on “How to add a layer definition” to the docs dir and linking it in the README.md

### Corresponding issue
Closes #124 

### New or changed dependencies
-

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [ ] ~My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing~
- [ ] ~I have commented my code~
- [ ] ~I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)~
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

### CODE REVIEW

File: docs/layer_adding.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+purposes and the ohsome API endpoint as well as filter for functionality.
+
+### Where to find information on how to define the ohsome API endpoint and filter?
+Information on how to define the endpoint and filter, which you need to state in your 
+layer definition, can be found in the 
+[ohsome API documentation](https://docs.ohsome.org/ohsome-api/stable/). There is a 
+section on the different 
+[API endpoints](https://docs.ohsome.org/ohsome-api/stable/endpoints.html#elements-aggregation)
+as well as a section on how to use and combine the 
+[filter](https://docs.ohsome.org/ohsome-api/v1/filter.html).
```
Comment (joker234) [inline docs/layer_adding.md:19]:
```suggestion
[filter](https://docs.ohsome.org/ohsome-api/stable/filter.html).
```

**Review by matthiasschaub — COMMENTED** [review]
Please have a look at my PR #155

File: docs/layer.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Layer
+
+A layer describes the request which should be made to the 
+[ohsome API](https://docs.ohsome.org/ohsome-api/v1/). Each layer is representative of a 
+specific set of features, aggregated information or user statistics derived from the 
+OpenStreetMap database. Each layer is defined by the ohsome API `endpoint` and 
+parameters for the `filter`. In addition each layer definition has a key, name and 
+description. Layer definitions are written down as YAML file at 
+`workers/ohsome_quality_analyst/ohsome/layer_defintions.yaml`
```
Comment (joker234) [inline docs/layer.md:9]:
```suggestion
`workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml`
```

File: docs/layer.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  https://docs.ohsome.org/ohsome-api/stable/endpoints.html
+- ohsome API documentation on the `filter` parameter: 
+  https://docs.ohsome.org/ohsome-api/stable/filter.html
+
+Second translate the query parameters into a layer definition and extent the file 
+`workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml`.
+
+Thirdly specify for which indicator class this layer definition is a valid input. Add 
+those indicator/layer combinations to the `INDICATOR_LAYER` tuple in the 
+`workers/ohsome_quality_analyst/utils/defintions.py` module. The tuple consists of the
```
Comment (joker234) [inline docs/layer.md:38]:
```suggestion
`workers/ohsome_quality_analyst/utils/definitions.py` module. The tuple consists of the 
```

File: docs/layer.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Layer
+
+A layer describes the request which should be made to the 
+[ohsome API](https://docs.ohsome.org/ohsome-api/v1/). Each layer is representative of a
```
Comment (joker234) [inline docs/layer.md:4]: One of the two:
```suggestion
[ohsome API](https://docs.ohsome.org/ohsome-api/stable/). Each layer is representative of a 
```
```suggestion
[ohsome API](https://api.ohsome.org). Each layer is representative of a 
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,7 +6,12 @@
 
 - Add API-endpoints to list indicators, reports, layers, datasets and feature id fields ([#106])
 
+### Other Changes
+
+- Added documentation on "How to add a layer definitions" ([#141])
```
Comment (joker234) [inline CHANGELOG.md:11]:
```suggestion
- Added documentation on “How to add a layer definitions” ([#141])
```

**Review by joker234 — APPROVED** [review]

File: docs/layer.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+Second translate the query parameters into a layer definition and extent the file 
+`workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml`.
+
+Thirdly specify for which indicator class this layer definition is a valid input. Add 
+those indicator/layer combinations to the `INDICATOR_LAYER` tuple in the 
+`workers/ohsome_quality_analyst/utils/definitions.py` module. The tuple consists of the 
+indicator class name and the layer definitions key as strings (E.g. 
+`("GhsPopComparisonBuildings", "building_count")`). If the specification was 
+successfully added to the tuple it is shown in the return of the command 
+`oqt list-layers`.
```
Comment (matthiasschaub) [inline docs/layer.md:42]:
```suggestion
`oqt list-layers`.

At last run `oqt create-inidicator --layer-name new-layer [...]` to check if the new layer can be used to make requests to the ohsome API and create an indicator successfully.
```

File: docs/layer.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+At last run `oqt create-inidicator --layer-name new-layer [...]` to check if the new layer can be used to make requests to the ohsome API and create an indicator successfully.
```
Comment (joker234) [inline docs/layer.md:44]:
```suggestion
At last run `oqt create-indicator --layer-name new-layer [...]` to check if the new layer can be used to make requests to the ohsome API and create an indicator successfully.
```

**Review by joker234 — APPROVED** [review]

**Pull request merged**